github/Installation.py

```diff
@@ -77,34 +152,92 @@ def target_type(self):
         """
         return self._target_type.value
 
+    @property
+    def updated_at(self):
+        """
+        :type: datetime.datetime
+        """
+        return self._updated_at.value
+
     def get_repos(self):
```

Comment:
I also don't think this method belongs in the `Installation` object as that is a `NonCompletable` and also the API call does not depend on any of the Installation attributes. It should be moved to the `github.Github.get_installation_repos`.

Reply:
Class `github.Github` is already cluttered with lots of class specific methods.

Why is `Github.get_installation_repos` better than `installation.get_repos`? With the latter it is clear that this requires an installation auth, while `Github` can be used with lots of different authentications, but that method works only with an installation auth.